server/db/models/order.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cartItem = new mongoose.schema(
+	qty: Number,
+	product: { type: Schema.Types.ObjectId, ref: 'Product' },
+	color: String
+);
+
+var address = new mongoose.schema(
```

Comment:
we created an address model last night. i'm thinking that the address used in the orders should be a reference to the address stored with a customer profile. we can use the model below or the other, but i think this should be a reference to the user address